cmd/geth/dbcmd.go

```diff
@@ -269,44 +322,125 @@ func confirmAndRemoveDB(database string, kind string) {
 	}
 }
 
-func inspect(ctx *cli.Context) error {
-	var (
-		prefix []byte
-		start  []byte
-	)
-	if ctx.NArg() > 2 {
-		return fmt.Errorf("max 2 arguments: %v", ctx.Command.ArgsUsage)
+func ancientInspect(ctx *cli.Context) error {
+	stack, _ := makeConfigNode(ctx)
+	defer stack.Close()
+
+	db := utils.MakeChainDatabase(ctx, stack, true, true)
+	defer db.Close()
+	return rawdb.AncientInspect(db)
+}
+func hbss2pbss(ctx *cli.Context) error {
+	if ctx.NArg() != 1 {
+		return fmt.Errorf("required arguments: %v", ctx.Command.ArgsUsage)
 	}
-	if ctx.NArg() >= 1 {
-		if d, err := hexutil.Decode(ctx.Args().Get(0)); err != nil {
-			return fmt.Errorf("failed to hex-decode 'prefix': %v", err)
-		} else {
-			prefix = d
-		}
+
+	stack, _ := makeConfigNode(ctx)
+	defer stack.Close()
+
+	db := utils.MakeChainDatabase(ctx, stack, false, false)
+	db.Sync()
+	defer db.Close()
+
+	config := trie.HashDefaults
+	triedb := trie.NewDatabase(db, config)
+	triedb.Cap(0)
+	fmt.Println("triedb scheme: ", triedb.Scheme())
```

Comment:
log is better, consistent with other cmds.

Reply:
and ditto to other `fmt.Println`

Reply:
fixed.